tests/test_buildspec.yaml

```diff
@@ -0,0 +1,13 @@
+version: 0.2
```

Comment:
what is this file for?

Reply:
file used to run the tests in cloud pipeline.
I can remove it if you want.

Reply:
nah no stress. But a comment at the top to help understand what might be useful